cypress/support/commands.ts

```diff
@@ -920,7 +920,8 @@ export const scriptsLoginWithFlags = (flags): Cypress.Chainable<any> => {
         })
         .then(() =>
           cy.reload().then(() => {
-            cy.getByTestID('page-title').contains('Data Explorer')
+            cy.getByTestID('tree-nav')
+            cy.getByTestID('data-explorer-page').should('exist')
```

Comment:
The page-title can change, when using a saved script. Therefore, did the proper test-id check for this fixture.